eval.go

```diff
@@ -2284,7 +2328,7 @@ func (e *callExpr) eval(app *app, args []string) {
 			switch app.ui.cmdPrefix {
 			case "!", "$", "%", "&":
 				app.ui.cmdPrefix = ":"
-			case ">", "rename: ":
+			case ">", "rename: ", "mkdir: ":
```

Comment:
Sorry, one more comment from me.

This will prevent the user from using backspace to 'cancel' out a `mkdir` command (i.e. pressing backspace when the input line looks like `mkdir: |`. I think it's better to allow the user to do this, similar to other commands like `find` and `mark-save`.

`rename` is different because it prefills the line with the original filename, and users will generally press and hold backspace to delete the original filename, see #1052 for more details.

Reply:
I also thought about that but decided to leave it because I wasn't sure.